src/controls/EnhancedThemeProvider/EnhancedThemeProvider.tsx

```diff
@@ -0,0 +1,91 @@
+import { ThemeProvider } from '@fluentui/react-theme-provider';
+import { getVariant, VariantThemeType } from "@fluentui/scheme-utilities";
+import { initializeIcons } from 'office-ui-fabric-react/lib/Icons';
+import { createTheme, getTheme, ITheme } from "office-ui-fabric-react/lib/Styling";
+import * as React from "react";
+import { useCallback, useEffect, useState } from 'react';
+import { fluentUITeamsDarkTheme } from '../../common/fluentUIThemes/FluentUITeamsDarkTheme';
+import { fluentUITeamsDefaultTheme } from '../../common/fluentUIThemes/FluentUITeamsDefaultTheme';
+import { fluentUITeamsHighContrastTheme } from '../../common/fluentUIThemes/FluentUITeamsHighContrastTheme';
+import { IEnhancedThemeProviderProps } from './IEnhancedThemeProviderProps';
+import { ThemeContext, useTheme } from '@fluentui/react-theme-provider';
+import * as telemetry from '../../common/telemetry';
+
+const getDefaultTheme = (): ITheme => {
```

Comment:
Is this the same as above?
![image](https://user-images.githubusercontent.com/19577724/167312812-32a2ddbd-8789-48ad-8a20-6493ee6cdf82.png)

Can you import it from the common folder?

Reply:
Yes! As the previous comment, I'll do a PR (I already made a big refactoring for the AdaptiveCardHost due to share more code from the AdaptiveCardHost and the future AdaptiveCardDesigner controls). If it's ok for you, I prefer to separare changes and have one PF for each controls... Ok for you?

Reply:
Sure, not a problem